UserShiftsApiService/UserShiftsApiService/Services/AddNewUserScheduleRequestService.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Linq;
+using UserShiftsApiService.Models;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore;
+using UserShiftsApiService.Entities;
+
+namespace UserShiftsApiService.Services;
+
+public class AddNewUserScheduleRequestService : IAddNewUserScheduleRequestService
+{
+    private readonly ShiftsSchedulingContext _dbContext;
+    private readonly IHttpContextAccessor _httpContextAccessor;
```

Comment:
nono. the service should not know what `HttpContext` is. the controller should resolve and send me the `userId` as a parameter